README.md

```diff
@@ -1 +1,88 @@
-# api-kanbansetup
\ No newline at end of file
+# Project Name 
+capstone: API-based webapp
+
+# Description of project.
+The JavaScript capstone project is about building  web application based on an external API. We selected an API that provides data about Meals and then built the webapp around it.
+
+## Project description video link:
+[video link](https://drive.google.com/file/d/1FxApYpw37fJ2XStkbNvisXXsrshisrMW/view?usp=sharing)
+
+
+## Project web demo link. 
+
+
+## Built With 
+- Html. 
+- Css. 
+- JS.
+- Webpark.
+- jest test
+
+Website link: 
+
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
+
+## Prerequisites
+ Vscode. 
+ must have git installed.
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location], or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone  https://github.com/jaamanu/Javascript-Capstone-Project.git
+- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+1. Install Node Modules
+
+   ```sh
+   npm install
```

Comment:
- [ ] Same issue as above, please use the example below.

  ![image](https://user-images.githubusercontent.com/45002654/198997067-2e6d441e-ae66-4be7-8de6-8fd1c5eae583.png)